glean-core/src/core/mod.rs

```diff
@@ -724,6 +724,15 @@ impl Glean {
             .insert(ping.name().to_string(), ping.clone());
     }
 
+    /// Gets a list of currently registered ping names.
+    ///
+    /// # Returns
+    ///
+    /// The list of ping names that are currently registered.
+    pub fn get_registered_ping_names(&self) -> Vec<&String> {
+        self.ping_registry.keys().collect()
```

Comment:
`&String` is an unuasl type to see in Rust.
This can be just `Vec<&str>`. Might need a `.map(String::as_str)` in there below.
```suggestion
    pub fn get_registered_ping_names(&self) -> Vec<&str> {
        self.ping_registry.keys().map(String::as_str).collect()
```